src/tsdl.mli

```diff
@@ -353,6 +353,7 @@ type rect
 
 module Rect : sig
   val create : x:int -> y:int -> w:int -> h:int -> rect
+  val modify : rect -> x:int -> y:int -> w:int -> h:int -> rect
```

Comment:
Regardless of acceptance this should return unit, the signature gives a wrong idea of what it is doing.